mozci/console/commands/push.py

```diff
@@ -365,6 +373,19 @@ def handle(self) -> None:
                 f"<info>Written stats for {len(self.pushes)} pushes in {output}</info>"
             )
 
+        if self.option("detailed-classifications"):
+            self.line("\n")
+            for push in self.pushes:
```

Comment:
Could you also print stats for all pushes instead of just push by push?

Reply:
Might be worth adding the "overall" stats to the email we send daily. Could be done in a follow-up.

Reply:
How should I build these "overall" stats and what should I display them?

Reply:
Basically:
X out of Y intermittent failures were correctly classified ("intermittent" by sheriff).
X out of Y intermittent failures were wrongly classified ("fixed by commit" by sheriff).
X out of Y intermittent failures were missed.
X out of Y real failures were correctly classified ("fixed by commit" by sheriff).
X out of Y real failures were wrongly classified ("intermittent" by sheriff).
X out of Y real failures were missed.

You can calculate the numbers by just summing them up from the individual pushes.